src/main/java/org/example/gameService/SaveGame.java

```diff
@@ -0,0 +1,65 @@
+package org.example.gameService;
+
+import org.example.model.GameData;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+
+public class SaveGame {
+
+    private VCSFileCreator vcsFileCreator;
```

Comment:
some of those field seem unnecessary, as they're not used anywhere else in the class

Reply:
I hope fixed. As above.